srcpkgs/chromium/template

```diff
@@ -1,23 +1,22 @@
 # Template file for 'chromium'
 pkgname=chromium
 # See http://www.chromium.org/developers/calendar for the latest version
-version=66.0.3359.170
-revision=2
+version=69.0.3497.100
+revision=1
 short_desc="Google's attempt at creating a safer, faster, and more stable browser"
-maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://www.chromium.org/"
-license="BSD"
+maintainer="Rasmus Thomsen <rasmus.thomsen@protonmail.com>"
+homepage="https://www.chromium.org/"
+license="BSD-3-Clause"
 distfiles="https://commondatastorage.googleapis.com/chromium-browser-official/${pkgname}-${version}.tar.xz"
-checksum=864da6649d19387698e3a89321042193708b2d9f56b3a778fb552166374871de
+checksum=e3391560e73e25fb4afc3f2dd5616607e2dbfc58aa88251a2c5d6b7096fe9e35
 
 only_for_archs="i686 x86_64 x86_64-musl"
 lib32disabled=yes
 nodebug=yes
 nopie=yes  # contains tools that are not PIE, enables PIE itself
 
-build_options="nacl clang"
-build_options_default="clang"
-desc_option_nacl="Enable NaCL support"
+build_options="clang"
+build_options_default=""
```

Comment:
if there are no default build_options you can just remove it

Reply:
Ah, yeah, I suppose this will be switched rather frequently, but I'll remove it.